projects/gsc/experiments/sampled_kwinners/kwinner_functions.py

```diff
@@ -0,0 +1,91 @@
+import torch
```

Comment:
Is this gsc/kwinners included intentionally? It doesn't directly tied to profiling dendrites.

Reply:
Yeah, this was an accident. Will update the PR to get rid of the other stuff.